src/diff_driver.h

```diff
@@ -8,7 +8,6 @@
 #define INCLUDE_diff_driver_h__
 
 #include "common.h"
-
```

Comment:
Nitpick: stray line deletion ("common.h" is special, IMHO it's best if it stands out).